.lando.yml

```diff
@@ -11,6 +11,7 @@ config:
   config:
     php: .lando/php.ini
     database: .lando/my.cnf
+    vcl: .lando/varnish.vcl
```

Comment:
Maybe I'm missing something but with a clean checkout I ran a destroy + start and still can't get the custom headers to appear when the config is here on the top level. When placing into `services → varnish → config → vcl` config loads just fine.

**Missing headers on dev tools**
![varnish-missing-vcl-header-lando](https://github.com/wunderio/drupal-project/assets/1128228/762fd9a5-875f-4a09-bf4b-dc7e8db812b8)

Reply:
Looks like works now. Thanks!
<img width="351" alt="Screenshot 2023-05-11 at 07 28 25" src="https://github.com/wunderio/drupal-project/assets/146800/46b73424-a11d-42f5-9212-e28e4feea6a7">